resources/strings/src/commonMain/kotlin/dev/sasikanth/rss/reader/resources/strings/ZhTwineStrings.kt

```diff
@@ -0,0 +1,152 @@
+package dev.sasikanth.rss.reader.resources.strings
+
+import cafe.adriel.lyricist.LyricistStrings
+
+@LyricistStrings(languageTag = Locales.ZH, default = true)
```

Comment:
Please mark `default` as false

Reply:
Sorry for the inconvenience, I've now set 'default' to 'false'. I'm actually new to coding and this is my first time using the pull requests feature. I'm not sure if this operation is correct. Please let me know if there are any mistakes.